zoo-css-challenge/style.css

```diff
@@ -85,45 +90,71 @@ footer {
   border-radius: 50%;
   display: inline-block;
   position: fixed;
-  bottom: 20px;
+  /* bottom: 20px; */
   right: 20px;
-
+  top: 20px;
   display: flex;
   justify-content: center;
   align-content: center;
   flex-direction: column;
+  transition: all 0.25s ease-in;
```

Comment:
```suggestion
```
We don't need to duplicate `transition` here as we already have it in the `:hover` below